examples/flutter_view/ios/Runner/MainViewController.m

```diff
@@ -36,13 +33,22 @@ - (void)prepareForSegue:(UIStoryboardSegue*)segue sender:(id)sender {
   }
 
   if ([segue.identifier isEqualToString:@"FlutterViewControllerSegue"]) {
-     self.flutterViewController = segue.destinationViewController;
-    [self.flutterViewController addMessageListener:self];
+    self.flutterViewController = segue.destinationViewController;
+
+    self.messageChannel = [FlutterMessageChannel messageChannelNamed:channel
```

Comment:
Se might be able to move channel initialization to the constructor, and make the property read-only